build/js/tests/bootstrap.js

```diff
@@ -1,3 +1,3 @@
 require('@babel/register')({
-  ignore: [/node_modules\/(?!@gdbots|lodash-es)/],
+  ignore: [/node_modules\/(?!@gdbots|@triniti|lodash-es)/],
```

Comment:
@gdbrown does this make sense? we are in triniti

Reply:
@efjacobson you're right, it makes no sense here